### PR TITLE
Handle no mutations

### PR DESCRIPTION
**Pull request by murphycj2**
I made these changes to handle the case for when I have ACCESS tumor samples that don't have any mutations AND also meets one of the following conditions:

- It has no DMP sample
- It has a DMP sample, but it has no mutations.

I successfully tested these changes on a project that had samples in both of the above categories.

Specifically, these changes allow me to run the compile_reads.R script successfully.

### CODE REVIEW

File: genotype_variants/create_duplex_simplex_dataframe.py [hunk 1]
```diff
@@ -67,9 +67,12 @@ def create_duplex_simplex_dataframe(simplex_dataframe, duplex_dataframe):
         exit(1)
 
     try:
```
Comment (rhshah) [inline genotype_variants/create_duplex_simplex_dataframe.py]: This try and except statements are for merging purpose, can you have them after the if shape statement

```
if shape>0:
   try:
        merge
   except:

else:
```

**Review by rhshah — CHANGES_REQUESTED** [review]
Thank you for working on this Charlie, I did not realize you were referring to this codebase. This will definitely help. Anyhow the place where you are adding the if shape statements can you add the try-except after the if as I showed in one example.

**Comment by murphycj2** [comment]
ok updated

**Review by rhshah — APPROVED** [review]
Thank you

**Pull request merged**